### PR TITLE
Create WebP Analysis command

### PR DESCRIPTION
**Pull request by jjgrainger**
* Adds the `wp webp-analysis run` command.
* Command exports to csv file in the mapped `./exports` folder
* Command prefixes csv files with unix `time()`, e.g `1661339116-webp-quality-analysis.csv`
* Command provides a `--limit=<limit>` option to set total number of attachments to analyze. Defaults to 500
* Command provides `--preview` option to preview results in ASCII table
* Add README instructions

### CODE REVIEW

File: mu-plugins/wp-cli-webp-analysis-command.php [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        // Write results to CSV file with timestamp.
+        $filename = plugin_dir_path( __FILE__ ) . "../exports/" . time() . "-webp-quality-analysis.csv";
+        $file     = fopen( $filename, 'w' );
+        write_csv( $file, $items, array_keys( $items[0] ) );
+
+        WP_CLI::success( sprintf( __( 'WebP analysis export created: %s', 'webp-quality-analysis' ), wp_basename( $filename ) ) );
+    }
+}
+
+if ( class_exists( 'WP_CLI' ) ) {
```
Comment (ankitrox) [inline mu-plugins/wp-cli-webp-analysis-command.php:121]:
```suggestion
if ( defined( 'WP_CLI' ) && WP_CLI ) {
```

File: mu-plugins/wp-cli-webp-analysis-command.php [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+            'posts_per_page' => $limit,
+        ];
+
+        $query = new WP_Query( $query_args );
+
+        if ( empty( $query->posts ) ) {
+            WP_CLI::error( __( 'No attachments found.') );
+        }
+
+        WP_CLI::line( sprintf( __( '%d attachments found.', 'webp-quality-analysis' ), count( $query->posts ) ) );
```
Comment (mukeshpanchal27) [inline mu-plugins/wp-cli-webp-analysis-command.php]:
```suggestion
        WP_CLI::line( sprintf( __( '%d attachments found.', 'webp-quality-analysis' ), $query->found_posts ) );
```

**Review by ankitrox — CHANGES_REQUESTED** [review]

**Review by mukeshpanchal27 — APPROVED** [review]
LGTM

File: mu-plugins/wp-cli-webp-analysis-command.php (hunk 1 above)
Comment (jjgrainger) [reply mu-plugins/wp-cli-webp-analysis-command.php:121]: @ankitrox this was added earlier in the file, please can review again?

**Review by ankitrox — APPROVED** [review]

**Pull request merged**